src/streams.cpp

```diff
@@ -11,6 +11,7 @@
 AutoFile::AutoFile(std::FILE* file, std::vector<std::byte> data_xor)
     : m_file{file}, m_xor{std::move(data_xor)}
 {
+    assert(m_xor.size() == 8);
```

Comment:
Guess the point of adding this assert is to prove that it doesn't break anything before switching to uint64?

Reply:
yes